File: src/main/java/com/spring/dto/response/PlaylistResponse.java
package com.spring.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class PlaylistResponse {
    private Long id;
    private String playlistName;
    private Float playTimeLength;
    private String releaseDate;
    private List<Long> songIdList;
    private List<String> songNameList;
    private List<Long> artistIdList;
    private List<String> artistNameList;
    private String imageUrl;
    private String description;
    private String status;
}

File: MSMA_App/app/src/main/java/vn/edu/usth/msma/data/dto/response/management/PlaylistResponse.kt
package vn.edu.usth.msma.data.dto.response.management

import com.google.gson.annotations.SerializedName

data class PlaylistResponse(
    @SerializedName("id") val id: Long,
    @SerializedName("playlistName") val playlistName: String,
    @SerializedName("playTimeLength") val playTimeLength: Float,
    @SerializedName("releaseDate") val releaseDate: String,
    @SerializedName("songIdList") val songIdList: List<Long>,
    @SerializedName("songNameList") val songNameList: List<String>,
    @SerializedName("artistIdList") val artistIdList: List<Long>,
    @SerializedName("artistNameList") val artistNameList: List<String>,
    @SerializedName("imageUrl") val imageUrl: String,
    @SerializedName("description") val description: String,
    @SerializedName("status") val status: String
)
